### ISSUE TITLE
ember inputs always render as disabled

### ISSUE THREAD

**Issue by mike-north** [ember-fastboot/ember-cli-fastboot#161]
When rendering inputs using the input helper, the disabled attribute in the input tag comes in stringified. This results in inputs *always* being rendered as disabled

## Markup 

```hbs
<p>
  <div>A W3C input (disabled is not present in attrs)</div>
  <input type="text" value='Hello Input!'>
</p>

<p>
  <div>A W3C input (disabled is present in attrs)</div>
  <input type="text" disabled value='Hello Input!'>
</p>

<p>
  <div>An Ember input (disabled is not specified)</div>
  {{input type="text" value='Hello Input!'}}
</p>

<p>
  <div>An Ember input (disabled=true)</div>
  {{input type="text" disabled=true value='Hello Input!'}}
</p>

<p>
  <div>An Ember input (disabled=false)</div>
  {{input type="text" disabled=false value='Hello Input!'}}
</p>
```

## Rendered HTML

![disabled inputs](https://raw.githubusercontent.com/mike-north/bug-fastboot-inputs/master/Screen%20Shot%202016-04-11%20at%203.38.11%20PM.png)

```html
<p>
  </p><div>A W3C input (disabled is not present in attrs)</div>
  <input type="text" value="Hello Input!">
<p></p>

<p>
  </p><div>A W3C input (disabled is present in attrs)</div>
  <input type="text" disabled="" value="Hello Input!">
<p></p>

<p>
  </p><div>An Ember input (disabled is not specified)</div>
  <input id="ember367" disabled="false" type="text" value="Hello Input!" class="ember-view ember-text-field">
<p></p>

<p>
  </p><div>An Ember input (disabled=true)</div>
  <input id="ember368" disabled="true" type="text" value="Hello Input!" class="ember-view ember-text-field">
<p></p>

<p>
  </p><div>An Ember input (disabled=false)</div>
  <input id="ember369" disabled="false" type="text" value="Hello Input!" class="ember-view ember-text-field">
```

*Simple project for reproducing:*
https://github.com/mike-north/bug-fastboot-inputs

* Clone it
* `npm i; bower i`
* `ember fastboot --serve-assets`

**Comment by mike-north** [comment]
According to [the W3C spec](https://www.w3.org/TR/html401/interact/forms.html#adef-disabled), disabled is referred to as a "boolean attribute". Maybe this concept of a non-string attribute has not been implemented in the fastboot simple DOM representation yet?

**Comment by rwjblue** [comment]
Confirm.  This likely needs some work in simple-dom (@krisselden may have an opinion here though).

**Comment by keithkelly** [comment]
Just curious if there is any update on this?

**Comment by jasonmit** [comment]
It was discussed in today's meeting but I had audio issues the whole time, but it is on the radar since it was on the agenda.

**Comment by keithkelly** [comment]
Actually just realized what might have been causing my problem.  I was not properly serving the static assets on the server.  Once I added `app.use('/assets', express.static('dist/assets'));` to my server files it seems like it is rendering it properly.

**Comment by asleroid** [comment]
I am running this issue as well.

It reproes when I run with "ember fastboot", with bunch of unresolved file paths for assets which explains why @keithkelly's fix works with express middleware. 

One example error for one asset is 
http://localhost:3000/assets/vendor.js Failed to load resource: the server responded with a status of 404 (Not Found)

I use the heroku buildpack without express middleware so this is still an issue for me.  

When I run with "ember fastboot --serve-assets" index.hbs gets rendered twice, but the last view that gets appended to the dom has inputs enabled and there are no 404s. 

I am new to fastboot so apologies for any extra confusion :)

**Comment by tomdale** [comment]
Can someone confirm this is fixed in the latest beta release?

**Comment by mike-north** [comment]
TL;DR - not sure, because I'm not sure what the expected behavior is.

Source of example is here
https://github.com/mike-north/fastboot-disabled-inputs

And if you run `ember build && ember fastboot` you'll see the same inputs disabled as reported in the original issue. I added some CSS to make it easier to see
<img width="849" alt="screen shot 2016-06-24 at 2 41 52 pm" src="https://cloud.githubusercontent.com/assets/558005/16347437/b9c57330-3a1a-11e6-976c-91fdf18048bb.png">

Of course, if you add the `--serve-assets` flag, ember will render the inputs correctly on the client side. When I originally reported the bug, I think client-side assets were not being served (resulting in the reported behavior, *even with `--serve-assets`*), however the original problem still exists as reported for the server-side-rendered html. Boolean attribute values are still serialized as strings.

**Comment by rwjblue** [comment]
The fix would have landed with Ember 2.7.0-beta.1. I submitted https://github.com/mike-north/fastboot-disabled-inputs/pull/1 to show that it works properly.

![screenshot with 2.7.0-beta.1](http://assets.rwjblue.com/snapshots/2016-06-25-12-58-37_FbInputs.png)

![screenshot of DOM with 2.7.0-beta.1](http://assets.rwjblue.com/snapshots/2016-06-25-13-00-47_FbInputs.png)
